src/app/reset-password/reset-password.component.html

```diff
@@ -52,4 +60,8 @@
       Sorry, there was an error saving your new password. Your link may have expired. Please <a href="https://www.thebiggive.org.uk/s/contact-us" target="_blank">contact us</a> if this message persists.
     </p>
 
-</div>
\ No newline at end of file
+    <div *ngIf="tokenValid === false"  class="error">
+      Sorry, there was an error changing your new password. Your link may have expired. Please <a href="https://www.thebiggive.org.uk/s/contact-us" target="_blank">contact us</a> if this message persists.
```

Comment:
@bdsl The user will have never tried to change their password when this message is shown, so let's remove that part:
```suggestion
      Sorry, your link may have expired. Please <a href="https://www.thebiggive.org.uk/s/contact-us" target="_blank">contact us</a> if this message persists.
```

Reply:
Yeah that's true. I think I was confused because I was thinking about the several different possible reasons for the error.